Lib/test/test_asyncio/test_sendfile.py

```diff
@@ -456,6 +456,8 @@ def test_sendfile_ssl_close_peer_after_receiving(self):
     # themselves).
     @unittest.skipIf(sys.platform.startswith('sunos'),
                      "Doesn't work on Solaris")
+    @unittest.skipIf(sys.platform == "win32",
+                     "It is flaky on Windows and needs to be fixed")  # TODO: bpo-41682
```

Comment:
nitpick: I'd probably include the bug number in the skip message in case that's all the viewer sees.

Reply:
I suggest to not include it in the skip message. Otherwise, https://bugs.python.org/issue41682 will be spammed with pull requests unrelated to the change, if a CI job fails with this message. The bot which mentions PR in bpo searchs for "bpo-xxx" anyway in a PR :-(

Reply:
Ouch!